frontends/api/src/test-utils/factories/learningResources.ts

```diff
@@ -129,6 +129,13 @@ const learningResourceOfferor: Factory<LearningResourceOfferor> = (
     code: faker.helpers.unique(faker.lorem.words),
     name: faker.helpers.unique(faker.lorem.words),
     channel_url: faker.internet.url(),
+    offerings: repeat(faker.lorem.word),
+    audience: repeat(faker.lorem.word),
+    formats: repeat(faker.lorem.word),
+    fee: repeat(faker.lorem.word),
+    certifications: repeat(faker.lorem.word),
+    content_types: repeat(faker.lorem.word),
+    more_information: faker.internet.url,
```

Comment:
This needs to be `faker.internet.url()`. In the `repeat` versions above, it's a function, b/c `repeat` calls the function. 

On line 125, I would suggest changing this to
```ts
const learningResourceOfferor: Factory<LearningResourceOfferorDetail> = (
  overrides = {},
) => {
```
because then TS would have caught this.